pyne/mesh.py

```diff
@@ -1452,6 +1501,22 @@ def tag_cell_fracs(self, cell_fracs):
                  size=1, dtype=float)
 
 
+    def tag_e_bins(self, e_bins):
```

Comment:
This is a nice convenience function, but it seems a little out of place, a little too specific, for this generic `Mesh` class. It might be better off in the location of the code being used?